docs/docs/Concepts/embedded-chat-widget.md

```diff
@@ -0,0 +1,271 @@
+    flow_id="dcbed533-859f-4b99-b1f5-16fce884f28f"
+  ></langflow-chat>
+</body>
+</html>
+```
+
+When embedded within HTML, this code becomes a responsive chatbot, powered by the basic prompting flow.
+
+![Default chat widget](/img/chat-widget-default.png)
+
+To configure your chat widget further, include additional props.
+
+All props and their types are listed in [index.tsx](https://github.com/langflow-ai/langflow-embedded-chat/blob/main/src/index.tsx).
+
+To add some styling to the chat widget, customize its elements with JSON:
+```html
+  <langflow-chat
+    host_url="https://c822-73-64-93-151.ngrok-free.app"
+    flow_id="dcbed533-859f-4b99-b1f5-16fce884f28f"
+    chat_window_style='{
+      "backgroundColor": "#1a0d0d",
+      "border": "4px solid #b30000",
+      "borderRadius": "16px",
+      "boxShadow": "0 8px 32px #b30000",
+      "color": "#fff",
+      "fontFamily": "Georgia, serif",
+      "padding": "16px"
+    }'
+    window_title="Custom Styled Chat"
+    height="600"
+    width="400"
+  ></langflow-chat>
+```
+
+To add a custom [session ID](/session-id) value and an API key for authentication to your Langflow server:
+```html
+<head>
+    <script src="https://cdn.jsdelivr.net/gh/logspace-ai/langflow-embedded-chat@main/dist/build/static/js/bundle.min.js"></script>
+</head>
+<body>
+  <langflow-chat
+    host_url="https://c822-73-64-93-151.ngrok-free.app"
+    flow_id="dcbed533-859f-4b99-b1f5-16fce884f28f"
+    api_key="YOUR_API_KEY"
+    session_id="YOUR_SESSION_ID"
+  ></langflow-chat>
+</body>
+</html>
+```
+
+The chat widget requires your flow to contain **Chat Input** and **Chat Output** components for the widget to communicate with it.
+Sending a message to Langflow without a **Chat Input** still triggers the flow, but the LLM warns you the message is empty.
+**Text Input** and **Text Output** components can send and receive messages with Langflow, but without the ongoing LLM "chat" context.
+
+## Embed the chat widget with React
+
+To use the chat widget in your React application, create a component that loads the widget script and renders the chat interface:
+
+1. Declare your web component and encapsulate it in a React component.
+
+```javascript
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      "langflow-chat": any;
+    }
+  }
+}
+
+export default function ChatWidget({ className }) {
+  return (
+    <div className={className}>
+      <langflow-chat
+        host_url="https://c822-73-64-93-151.ngrok-free.app"
+        flow_id="dcbed533-859f-4b99-b1f5-16fce884f28f"
+      ></langflow-chat>
+    </div>
+  );
+}
+```
+2. Place the component anywhere in your code to display the chat widget.
+
+For example, in this docset, the React widget component is located at `docs > src > components > ChatWidget > index.tsx`.
+It includes a script to load the chat widget code from CDN, and initialize the chat widget with props pointing to a Langflow server.
```

Comment:
I'd make a note to acknowledge you are just moving the above code into a file including the loader. Otherwise it just looks like we have essentially the same code block twice in a row